test/e2e-tests.sh

```diff
@@ -28,7 +28,7 @@
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 
 # Location of istio for the test cluster
-readonly ISTIO_YAML=./third_party/istio-1.0.0/istio.yaml
+readonly ISTIO_YAML=./third_party/istio-1.0.0/istio-lean.yaml
```

Comment:
This is currently unused, what's the motivation behind adding it here?

Reply:
I filed #1898 to use both files in our tests.

Reply:
Mainly we want to make sure we can work with more than one way to install Istio.

Reply:
Yes, I saw the bug and I agree. However, simply adding the var here won't accomplish anything...

Reply:
It switches us to using the new yaml in the integration test so we know it's working.

Reply:
Please ignore my comment, as I misread the change. You're using the lean istio config, not adding a new one. My apologies for the confusion.